control-plane/agents/src/bin/core/controller/reconciler/volume/garbage_collector.rs

```diff
@@ -159,7 +159,9 @@ async fn disown_unused_nexuses(
                 }
             }
 
-            nexus.warn_span(|| tracing::warn!("Attempting to disown unused nexus"));
+            nexus.info_span(|| {
+                tracing::info!("Attempting to disown unused nexus for vol : {}", vol_id)
+            });
```

Comment:
volume owner is already printed in this case IIRC

```suggestion
             nexus.info("Disowning volume from the nexus");
```

Reply:
more like `nexus.info("Disowning nexus from the volume");`

Reply:
Yes that's the right way around :D

Reply:
It actually remove `volume` of `owner` field of `NexusSpec`